src/features/hr/ica_estimator.py

```diff
@@ -0,0 +1,32 @@
+from base_estimator import BaseEstimator
+from pyVHR.methods.ica import ICA
+
+
+class ICAEstimator(BaseEstimator):
+
+    """Defines a wrapper for heart rate estimator
+    based in Idependent Component Analysis (ICA)
+    of remote photoplethysmography (rPPG) data
+    from pyVHR implementation.
```

Comment:
In PR #26 I didn't cared about docstring max line length (I believe python says 72?). So we should use 72 for the max length, and later I will change the merged code. We may also set another limit if needed.

Reply:
Interesting, I guess 72 it's fine, it may be helpful in docs generation